src/handler-context/README.md

```diff
@@ -0,0 +1,58 @@
+# `handler-context`
+
+This code is designed to help devs to write Lambda function handlers in a way which separates business logic from common input and output functionality
+
+Note: this documentation might be out of date. Please update it as you change the code!
+
+Here is an example of how to make a Lambda handler with this code:
+
+```ts
+export const handler = buildHandler({
+  envVars,
+  ConfigCache,
+  incomingMessageBodyTypeGuard,
+  businessLogic,
+  outputs,
+  withBatchItemFailures,
+});
+```
+
+## `envVars`
+
+`envVars` defines which environment variables the handler can use
+
+Should any specified variable not exist in the handler's execution environment when it runs, `buildHandler` will throw an error with a message which lists the missing variables
+
+`envVars` must be an array of strings of the type `EnvVarName` in `src/shared/utils/env.ts`, so add new strings there for any additional environment variables which you would like to use
+
+## `ConfigCache`
+
+`ConfigCache` defines which runtime config file data the handler can use from the private config repo
```

Comment:
Maybe mention that the various config options correspond to a config file in the di-btm-config project.